documentation/content/en/books/handbook/security/_index.adoc

```diff
@@ -1913,7 +1910,7 @@ To force the users to use keys only, the following option may be configured:
 
 [.programlisting]
 ....
-AuthenticationMethods publickey
```

Comment:
I'm gonna commit all except this, based on the information we have just before is correct, in this way we force users to make the login with the public key

Reply:
Thank you for your helpful review.  You're right that the Handbook in its current form is correct: I'm unsure why I believed otherwise.  I've pushed an updated version of my branch that contains only the one desirable commit.